packages/gatsby-codemods/src/transforms/gatsby-plugin-image.js

```diff
@@ -193,8 +193,9 @@ function processImportUsage(path, t, template, state) {
   const node = path.parent
 
   if (!t.isJSXOpeningElement(node)) {
+    path.node.name = `GatsbyImage`
     console.log(
-      `It appears you're using the image component for something this codemod does not currently support. You will need to change the reference in ${state.opts.filename} manually.`
+      `It appears you're extending or referencing the image component in some way. We've updated the reference, but you will want to verify ${state.opts.filename} manually.`
```

Comment:
Ah, answering my previous question! Are we abe to tell the difference between extends and references? It would be nice if we could warn them that it can't be extended and they should use composition instead.

Reply:
To a certain degree we can, but we might miss one or two.